### ISSUE TITLE
Customized dataset training with large synthesized dataset and small real dataset

### ISSUE THREAD

**Issue by PrinceCatt** [NVlabs/Deep_Object_Pose#415]
I synthesized data using Blender and trained for 200 epochs with 5k images.
When I use this weight to do inference on some other synthesized data it showed incomplete cuboid detection for some of the images while it correctly detected the cuboid for some. But what is causing this problem? There was no occlusion of the object. 

Based on the weights from synthesized data, I then trained with 300 real images (75 + augmentation). But when I then used it to predict the real images  in the training set, it also showed incomplete cuboid detection although some of corner points were recognized.

So may I ask what is causing this problem of incomplete? When I take a look at the cli output there was actually 8 points. Are there any recommendations on how to improve?

**Comment by nv-jeff** [comment]
Most of the DOPE training I've done has been with synthetic images, although we have found that a mixture of real and synthetic usually provides the best results.

You could be running into one of several problems:
- DOPE converges slowly.  You may need 1000s of epochs to train to train a network to the accuracy reported in the paper (about 70% w/ 1cm pose accuracy)
- You may need more data. 300 real + 5k synthetic is good, but more data in varied domains will help.
- The threshold for "detection" of a predicted cuboid point can be changed if you're willing to have more detections at the cost of lower accuracy.  Look in `config/config_pose.yaml`. At the bottom of the file are "config params for DOPE"; varying these (especially `thresh_points`) can loosen the detection criteria. I suggest you dig into the code to see how these parameters are used before experimenting.
